near/contracts/ft/src/lib.rs

```diff
@@ -157,6 +157,10 @@ impl FTContract {
                 env::panic_str(&format!("paylod1 formatting errro  len = {}", p.len()));
             }
         } else {
+            if fee != 0 {
+                env::panic_str("Payload3 does not support fees");
```

Comment:
Is this only hit for Payload3, as the error indicates?

Reply:
Yup, this is the P3 case..